connexion/apis/aiohttp_api.py

```diff
@@ -30,7 +30,7 @@
 def oauth_problem_middleware(request, handler):
     try:
         response = yield from handler(request)
-    except (OAuthProblem, OAuthScopeProblem) as oauth_error:
+    except (OAuthProblem, OAuthScopeProblem, OAuthResponseProblem) as oauth_error:
```

Comment:
`OAuthResponseProblem` is a subclass of `OAuthProblem`.. I'm confused why this is needed.

Reply:
Sorry, I forgot to close it after seeing that [PR 830](https://github.com/zalando/connexion/pull/830 ) was also solving that.